README.md

```diff
@@ -1 +0,0 @@
-# ExampleRepo
```

Comment:
Add a brief outline for the implemented test cases in order to understand what contains this repository.